### PR TITLE
CPO on perftest

### PR DESCRIPTION
**Pull request by smathangi**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by hemantt — APPROVED** [review]

File: k8s/namespaces/cpo/case-payment-orders-api/perftest.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
```
Comment (danlysiak) [inline k8s/namespaces/cpo/case-payment-orders-api/perftest.yaml:1]: Need to add to kustomize too?

Comment (smathangi) [reply k8s/namespaces/cpo/case-payment-orders-api/perftest.yaml:1]: it's there in the common place https://github.com/hmcts/cnp-flux-config/blob/cpo_perftest/k8s/namespaces/cpo/kustomization.yaml

Comment (danlysiak) [reply k8s/namespaces/cpo/case-payment-orders-api/perftest.yaml:1]: Sorry, I meant in the common-overlay - e.g. https://github.com/hmcts/cnp-flux-config/blob/cpo_perftest/k8s/demo/common-overlay/cpo/kustomization.yaml

**Review by KleoG — DISMISSED** [review]

File: k8s/namespaces/cpo/case-payment-orders-api/perftest.yaml (hunk 1 above)
Comment (smathangi) [reply k8s/namespaces/cpo/case-payment-orders-api/perftest.yaml:1]: aha.. you're saying entry of case-payment-orders-api.yaml in the kustomize. But it works without that for other envs 🤔

Comment (smathangi) [reply k8s/namespaces/cpo/case-payment-orders-api/perftest.yaml:1]: added now. Thanks @danlysiak

**Review by danlysiak — APPROVED** [review]

**Review by adusumillipraveen — COMMENTED** [review]
You need an entry here : https://github.com/hmcts/cnp-flux-config/blob/27c02bee13b2b1bf1cb21d4217625923051a7580/k8s/perftest/common-overlay/kustomization.yaml#L12

**Comment by smathangi** [comment]
> You need an entry here :
> 
> https://github.com/hmcts/cnp-flux-config/blob/27c02bee13b2b1bf1cb21d4217625923051a7580/k8s/perftest/common-overlay/kustomization.yaml#L12

Thanks @adusumillipraveen  done in c39f08bf4e1700b168812135ad1e8d5c85389765

**Review by danlysiak — APPROVED** [review]

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**